aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2206,6 +2206,21 @@ nimble:
   tags:
     product_categories: {}
   version: '' # no longer published
+oam:
+  api_ref: OAM/latest/APIReference/Welcome.html
+  blurb: ''
+  expanded:
+    long: Amazon CloudWatch Observability Access Monitor
+    short: CloudWatch Observability Access Monitor
+  guide:
+    subtitle: Prescriptive Guidance
+    url: rescriptive-guidance/latest/patterns/centralize-monitoring-by-using-amazon-cloudwatch-observability-access-manager.htmlp
```

Comment:
There's a typo in this URL. Somehow the 'p' from prescriptive got moved to the end... 😆